test/e2e/collect-ci-artifacts.sh

```diff
@@ -24,3 +25,8 @@ for command in "${commands[@]}"; do
     COMMAND_OUTPUT_FILE=${TEST_ARTIFACTS_DIR}/${command// /_}
     kubectl -n ${TEST_NAMESPACE} ${command} >> "${COMMAND_OUTPUT_FILE}"
 done
+
+echo "Deleting any empty files in the ${TEST_ARTIFACTS_DIR} directory"
+# Note: delete any files less than 83 kilobytes as this should account for
+# any empty files, and files that contain zero length arrays of `kind: List`.
+find "${TEST_ARTIFACTS_DIR}" -type f -size -83k -delete
```

Comment:
Hmm I realize we should probably torch empty directories as well. It looks like find also has support for identifying empty directories so it should just be another command.